### ISSUE TITLE
No database

### ISSUE THREAD

**Issue by Drippers** [AbenezerMamo/Steam-Group-Inviter#1]
whell i cant find the database can you please upload it ? :)

**Comment by hackerpedro** [comment]
Yes, the database is missing, help is appreciated!

**Comment by msburgess3200** [comment]
Bump. No database. Though, if I get time, I will paste a database.

**Comment by T0b1-iOS** [comment]
yep it would be cool if we could at least get a plan how the database should look like

**Comment by jhilgenberg** [comment]
Still no database and no documentation? Would be awesome if someone could update it.
